api/opencl_runtime_layer.asciidoc

```diff
@@ -6388,6 +6435,11 @@ Otherwise, it returns one of the following errors:
     the return type specified in the
     <<mem-info-table, Memory Object Queries>> table
     and _param_value_ is not `NULL`.
+ifdef::cl_ext_buffer_device_address[]
+  ** Returned for the {CL_MEM_DEVICE_ADDRESS_EXT} query if
+     the {cl_ext_buffer_device_address_EXT} is not supported or if the
+     buffer was not allocated with {CL_MEM_DEVICE_PRIVATE_ADDRESS_EXT}.
```

Comment:
I'd consider returning `CL_INVALID_OPERATION` instead of `CL_INVALID_VALUE` in the case where the buffer was not allocated with `CL_MEM_DEVICE_PRIVATE_ADDRESS_EXT`.  Rationale is: in this scenario the value is valid, it is just invalid in combination with this specific buffer.